translations/en-GB.json

```
@@ -278,14 +278,14 @@
   "RELATIVE_MAX_AGE_TIME": "Newer than {time}",
   "INVALID_DATE_AUTOCOMPLETE": "Invalid Date",
   "INVALID_DATE_AUTOCOMPLETE_EDIT": "No Date (unset)",
-  "INVALID_DATE": "An invalid date was provided. Make sure your using the right format (dd/mm/yyyy for everyone, mm/dd/yyyy if you live in the US).",
+  "INVALID_DATE": "An invalid date was provided. Make sure you're using the right format (dd/mm/yyyy for everyone, mm/dd/yyyy if you live in the US).",
   "INVALID_AGE_AUTOCOMPLETE": "Invalid Age",
   "INVALID_AGE_AUTOCOMPLETE_EDIT": "No Age (unset)",
-  "INVALID_AGE": "An invalid age was provided. You can provide your age in seconds, minutes, hours, days, weeks, months (30d) and years. (example: `1 day 12 hours`)",
+  "INVALID_AGE": "An invalid age was provided. You can provide your age in seconds, minutes, hours, days, weeks, months (30d), and years. (example: `1 day 12 hours`)",
   "PLEASE_INPUT_FILTER_OPTIONS": "Please input some options for this filter.",
   "CANNOT_HAVE_EMPTY_FILTER": "A message filter can't exist without any options.",
   "SUCCESS_CREATE_FILTER": "Successfully created a message filter **{name}** based on the message's {type, select, 0 {content (and embed content)} 1 {attachments, links and embeds} other {age}}, which applies to {appliesTo, plural, =0 {everyone} =1 {1 user/role} other {# users/roles}} and does not apply to {doesNotApplyTo, plural, =1 {1 user/role} other {# users/roles}}.",
-  "TO_ADD_FILTER": "To enable this filter for messages across the whole server, run {filtersAdd}. If you want this filter to only apply for a set of channels, try creating a channel override with {overridesCreateChannel} and then add this filter to the newly created override with {filtersAdd}.",
+  "TO_ADD_FILTER": "To enable this filter for messages across the whole server, run {filtersAdd}. If you want this filter to only apply to a set of channels, try creating a channel override with {overridesCreateChannel} and then add this filter to the newly created override with {filtersAdd}.",
```

```suggestion
  "TO_ADD_FILTER": "To enable this filter for messages across the whole server, run {filtersAdd}. If you want this filter to only apply for a set of channels, try creating a channel override with {overridesCreateChannel} and then add this filter to the newly created override with {filtersAdd}.",
```
Doesn't feel right - Filters are being applied _in_ the channel or for messages of the channel, not necessarily to the channel itself